master/static/srv/get_experiments.sql

```diff
@@ -1,10 +1,15 @@
 WITH filtered_exps AS (
     SELECT
         e.id AS id,
+        e.config->>'name' AS name,
         e.config->>'description' AS description,
         e.config->'labels' AS labels,
         e.config->'resources'->>'resource_pool' AS resource_pool,
         e.config->'searcher'->'name' as searcher_type,
+        CASE
```

Comment:
I'm confused by this, if the notes as '' then they're not omitted right, they're just not there? wouldn't this be the query to always just return 'omitted' if we're not going to return them from list endpoints?

Reply:
I used this to encode whether the experiment has a note or not but we probably don't want users to rely on that 🤔  if so we can just set them all to 'omitted'

Reply:
Ah I get it now, duh

Reply:
keep?